modules/eks/actions-runner-controller/README.md

```diff
@@ -28,94 +28,101 @@ components:
+            webhook_driven_scaling_enabled: true
             pull_driven_scaling_enabled: false
             labels:
               - "Ubuntu"
               - "core-otto"
-          app-runner:
-            type: "repository" # can be either 'organization' or 'repository'
-            dind_enabled: true # If `true`, a Docker sidecar container will be deployed
-            # To run Docker in Docker (dind), change image from summerwind/actions-runner to summerwind/actions-runner-dind
-            image: summerwind/actions-runner-dind 
-            scope: "example/app"
-            scale_down_delay_seconds: 300
-            min_replicas: 1
-            max_replicas: 2
-            busy_metrics:
-              scale_up_threshold: 0.75
-              scale_down_threshold: 0.25
-              scale_up_factor: 2
-              scale_down_factor: 0.5
-            resources:
-              limits:
-                cpu: 2
-                memory: 6Gi
-                ephemeral_storage: "10Gi"
-              requests:
-                cpu: 0.5
-                memory: 1Gi
-            storage: "10Gi"
-            webhook_driven_scaling_enabled: false
-            pull_driven_scaling_enabled: false
-            labels:
-              - "Ubuntu"
-              - "core-otto"
-
 ```
 
-### Creating Github Tokens
+### Generating Required Secrets
+
+Generate the following secrets to be stored with AWS SSM:
 
-Ensure that the required tokens are created in AWS SSM.
-1. The `github_token` saved under `var.ssm_github_token_path`. The value should be a PAT with the scope outlined in [this document](https://github.com/actions-runner-controller/actions-runner-controller#deploying-using-pat-authentication).
-2. If using the Webhook Driven autoscaling (recommended), include the key `github_webhook_secret_token` saved under `var.ssm_github_webhook_secret_token_path`. Set to a random string you will use as the Secret when creating the webhook in GitHub.
+1. The key `github_token` and the value a PAT with the scope outlined in [this document](https://github.com/actions-runner-controller/actions-runner-controller#deploying-using-pat-authentication).
+2. If using the Webhook Driven autoscaling (recommended), include the key `github_webhook_secret_token` set to a random string you will use as the Secret when creating the webhook in GitHub.
 Generate the string using 1Password (no special characters, length 45) or by running
 ```bash
 dd if=/dev/random bs=1 count=33  2>/dev/null | base64
 ```
 
+### Setting AWS SSM Parameters 
+
+AWS SSM is used to store and retrieve the required secrets. Create a SSM Parameter for each secret generated with the "Generating Required Secrets" step above, and set the following variables to those Parameter paths:
+
+```
+ssm_github_token_path: "/github/acme/github_token"
+```
+
+And if using Webhook Driven autoscaling:
+```
+ssm_github_webhook_secret_token_path: "/github/acme/github_webhook_secret_token"
+```
+
```

Comment:
These 2 sections need to be combined and the whole thing re-written. That can be done in a future PR.

Reply:
Added a quick merge for now